Quaver.Shared/Screens/Edit/UI/AutoMods/EditorAutoModPanelContainer.cs

```diff
@@ -73,7 +73,8 @@ public EditorAutoModPanelContainer(EditScreen screen)
             Panel = new EditorAutoModPanel(Screen.WorkingMap, mapset)
             {
                 Parent = this,
-                Alignment = Alignment.MidRight
+                Alignment = Alignment.MidRight,
+                Container = this
```

Comment:
This should be passed into the constructor instead. Without this line, the entire game will crash when clicking the button because of a null reference, so you may as well force it to be set.

Reply:
Passing it as a constructor now, and force it to be set. I avoided doing that because ``AutoModTestScreenView`` also created a new panel and i didn't know what to pass, decided to pass ``Container as EditorAutoModPanelContainer`` and it work, hopefully that is the right way to do it

Reply:
That technically is incorrect because the `Container` in the test is for the screen rather than the automod container (which is very specific to the editor). Doesn't matter too much for the test though. This is ok.